docs/schemas/resource/IfcPresentationAppearanceResource/Entities/IfcIndexedPolygonalTextureMap.md

```diff
@@ -0,0 +1,16 @@
+# IfcIndexedPolygonalTextureMap
+
+The _IfcIndexedPolygonalTextureMap_ provides the mapping of the 2-dimensional texture coordinates to a set of polygonal bounded faces onto which it is mapped. It is used for mapping the texture to faces of an _IfcPolygonalFaceSet_. Such faces may have inner loops.
+
+The _TexCoords_ defined at supertype _IfcIndexedTextureMap_ are a two-dimensional list of texture coordinates provided by two parameter values. Each member of the set of _TexCoordIndices_ defines an unbounded index list corresponding to the vertices of the assigned polygonal face that points to the texture coordinates in _TexCoords_ for that vertex.
```

Comment:
"Each member of the set of _TexCoordIndices_ defines an unbounded index list corresponding to the vertices of the assigned polygonal face that points to the texture coordinates in _TexCoords_ for that vertex."

This sentence is really hard to follow for me. Can we make it simpler somehow by breaking it up into several sentences? Like in bullets:

- A polygonal face is defined as a sequence of indices to points
- To provide textures coordinates a corresponding sequence is provided with indices to texture coordinates
- ...

Reply:
I make a trial.

The set of _TexCoordIndices_ provides the texture coordinates for the list of _IfcIndexedPolygonalFace_ refered to via _MappedTo.Faces_. The texture coordinates at each _IfcTextureCoordinateIndices_ correspond to the vertex points at the referenced _IfcIndexedPolygonalFace_. See also Figure 1 for further explanation.